content/doc/administrate/cache.md

```diff
@@ -57,3 +57,19 @@ If you already have a configuration for an older version of varnish, you can rea
 We provide some [examples of Varnish configuration files](https://GitHub.com/CleverCloud/varnish-examples) that you can
 use for your application. Create a `/clevercloud` folder at the root of your application if it does not exist,
 rename the file to `varnish.vcl` and move it in the `/clevercloud` folder.
+
+## Varnish with a monorepo
+
+If you use a monorepo, you may want to use Varnish for only some of the applications inside it.  
+If you have a `/clevercloud/varnish.vcl` file at the root of your monorepo, all of your applications automatically start using Varnish.
+
+To resolve this issue, you can create a symlink during the deployments.  
+First, you need to put your `varnish.vcl` file anywhere but at the root of your monorepo.  
+Then, you need to create a symlink inside a `CC_PRE_BUILD_HOOK`.  
+Here is an example :
+
+```bash
+CC_PRE_BUILD_HOOK="mkdir $APP_HOME/clevercloud; ln -s $APP_HOME/path/to/your/file/varnish.vcl $APP_HOME/clevercloud/varnish.vcl"
+```
+
+This way, you only need to add the `CC_PRE_BUILD_HOOK` when you want your app to use Varnish. If you don't, the application won't use it.
```

Comment:
```suggestion
Then add the `CC_PRE_BUILD_HOOK` in the variables of the app that needs to use Varnish. If you don't add this variable, the application won't use it.
```

It doesn't seem clear to me how you would handle the monorepo deployment. Many readers might think that the whole monoprepo containing several apps will run on a single runtime and might not understand  the variable instruction.

It looks like you are thinking about deploying a monorepo divided in several runtimes, using the variable `APP_FOLDER`, it that right?

Reply:
This is indeed the case where someone has multiple app, sometime on different runtime and use `APP_FOLDER` to decide which app should be used.
For exemple, if a user has three different app in his monorepo, but only one need to use varnish, he needs to select that app with `APP_FOLDER` and follow the instructions in this PR.

If there was a variables like `CC_ENABLE_VARNISH=true`, the problem would be easily resolved. But right now, there is not a lot of way around that problem.